hibike/hibike_tester.py

```diff
@@ -0,0 +1,35 @@
+import hibike_process
+from multiprocessing import Process, Pipe, Queue
+
+class Hibike:
+
+    def __init__(self):
+
```

Comment:
Same thing here: no point for blank line (unless it's to remind yourself that you need a docstring... in which case you should write a docstring)